api/import_export/import_.py

```diff
@@ -0,0 +1,30 @@
+import os
+import uuid
+
+import boto3
+from django.core.management import call_command
+
+
+class OrganisationImporter:
+    def __init__(self, s3_client=None):
+        self._s3_client = s3_client or boto3.client("s3")
+
+    def import_organisation(self, s3_bucket: str, s3_key: str) -> None:
+        """
+        Import an organisation from a json file containing the django fixtures
+        as exported by the `export` module in this package.
+
+        This code essentially acts as a wrapper to the django loaddata management
+        command to grab a file from S3 and read from that.
+        """
+        obj = self._s3_client.get_object(Bucket=s3_bucket, Key=s3_key)
+        file_path = f"/tmp/{uuid.uuid4()}.json"
```

Comment:
I wonder if we can use: https://docs.python.org/3.8/library/tempfile.html#module-tempfile?

Reply:
I tried that but couldn't get it to work... I'm going to try again as I've changed a few things since I tried.

Reply:
Ok, yeah, so the issue is that loaddata expects the full path to a file or the name of a fixture. 

I'd hope to be able to do something like this: 

```python
        with NamedTemporaryFile() as f:
            f.write(obj["Body"].read())
            call_command("loaddata", f.name, format="json")
```

This results in the following error, however: 

```
django.core.management.base.CommandError: No fixture named 'tmpz0us6zlg' found.
```

I guess because the file doesn't actually exist in the file system at 'tmpz0us6zlg' when loaddata goes looking for it. As far as I can tell, there is no way to just pass a file object directly to the loaddata command which is quite frustrating. I could try and replicate the code from that command but it's kind of a monster (https://github.com/django/django/blob/main/django/core/management/commands/loaddata.py#L139) so I'd rather not. 

I have, however, improved the code slightly (and fixed a bug in the actual management command).